### DISCUSSION TITLE
Key_Calc - remapping possible?

### DISCUSSION THREAD

**Discussion by AndyFx12** [FreeRDP/FreeRDP#9988] [Q&A]
Hi!

I've got a german keyboard with special key "Calculator". This key won't be send to remote server with xfreerdp 2.9.0 (Linux Mint 21.3). This key works on native Cinnamon-Desktop and is reported via showkeys as keycode 140.

On xfreerdp 3.4.0 from flatpak, CTRL+ALT+RETURN don't work to leave fullscreen. WIN+RSHIFT+ENTER shows a black screen.
And my special key don´t work at 3.4.0, too.

Does anyone know how to remap/use this key to open remote calculator?

regards, Andy

**Comment by akallabeth** [comment]
@AndyFx12 
1. with `flatpak` you use `sdl-freerdp` which has a different hot key (`right shift` - configurable via config file though)
2. special keys are currently not mapped for the sdl client, patches welcome! (check `sdl_kbd.cpp` `static const scancode_entry_t map[]`)
3. if you want to run `xfreerdp` enable `X11`, disable `X11 fallback` with `flatseal` and run `flatpak run --command=xfreerdp com.freerdp.FreeRDP`
